pub-critic/src/app/components/navigation/Navigation.module.scss

```diff
@@ -0,0 +1,118 @@
+.Navigation {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    position: fixed;
+    top: 0;
+    padding: 20px 5vw;
+    background-color: black;
+    z-index: 100;
+    height: 70px;
+    @media (max-width: 768px) {
+        padding: 20px 15px;
+    }
+    .Name {
+        font-size: 0.7vw;
+        font-weight: 500;
+        color: white;
+        transition: color 0.3s ease-in-out;
+        &:hover {
+            color: #9ca3af;
+            cursor: pointer;
+        }
+        @media (max-width: 768px) {
+            font-size: 1.5vw;
+        }
+    }
+    .Search {
+        width: 32vw;
+        border-radius: 10px;
+        border: 1.5px solid #d1d5db;
+        align-items: center;
+        display: flex;
+        min-height: 30px;
+        padding: 10px 20px;
+        transition: border 0.3s ease-in-out;
+        @media (max-width: 768px) {
+            width: 60vw;
+        }
+        &:hover {
+            border: 1.5px solid #9ca3af;
+        }
+        &:focus-within {
+            border: 1.5px solid #fff;
+        }
+        input {
+            width: 90%;
+            height: 100%;
+            border: none;
+            outline: none;
+            font-size: 0.7vw;
+            font-weight: 500;
+            color: white;
+            background-color: transparent;
+            &::placeholder {
+                color: #d1d5db;
+                font-size: 0.7vw;
+                font-weight: 500;
+                @media (max-width: 768px) {
+                    font-size: 2.5vw;
+                }
+            }
+            @media (max-width: 768px) {
+                font-size: 2.5vw;
+            }
+        }
+    }
+    .Account {
+        font-size: 0.4vw;
+        font-weight: 500;
+        color: white;
+        transition: color 0.3s ease-in-out;
+        @media (max-width: 768px) {
+            display: none;
+        }
+        &:hover {
+            color: #9ca3af;
+        }
+    }
+    .Menu {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        align-items: center;
+        width: 10vw;
+        .ImageContainer {
+            border-radius: 50%;
+            border: 1.5px solid #d1d5db;
+            width: 1.5vw;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            height: 1.5vw;
+            @media (max-width: 768px) {
+                width: 6vw;
+                height: 6vw;
+                > img {
+                    width: 5vw;
+                    height: 5vw;
+                }
+            }
+            &:hover {
+                border: 1px solid #9ca3af;
+                cursor: pointer;
+            }
+        }
+        .MenuIcon {
+            border-radius: 50%;
+            width: 1vw;
+            height: 1vw;
+            color: white;
+            &:hover {
+                color: #9ca3af;
```

Comment:
use variables for colors

Reply:
:white_check_mark: created a variable for every color used